maint/clmake

```diff
@@ -0,0 +1,251 @@
+#! /usr/bin/env perl
+##
+## Copyright (C) by Argonne National Laboratory
+##     See COPYRIGHT in top-level directory
+##
+
+# Set Defaults
+$debug = 0;
+$debugDir = 0;
+$gDebugOther = 0;
+
```

Comment:
Can you add a comment in the commit describing what this script does and why is needed?

Reply:
Done.